cmd/zapi/cmd/get/get.go

```diff
@@ -25,8 +26,30 @@ var Get = &charm.Spec{
 	Name:  "get",
 	Usage: "get [options] <search>",
 	Short: "perform zql searches",
-	Long:  "TODO",
-	New:   New,
+	Long: `
+zapi get issues search requests to the zqd search service.
+
+The -from and -to options specify a time range.  If not provided, the entire
+space is searched.
+
+By default, the service streams results in native zng and the zapi client
+converts the results to the format specified by -f.
+
+Alternatively, the -e option can specify a different encoding to be used by
+the server, in which case, the output is not converted by -f.  These two options
+cannot be used together.
+
+Statistics and warnings can be displayed on stderr, which can be useful
+for long running jobs.
+
+Everything is flow-controlled end to end, so if the client host, network,
```

Comment:
flow control 🤩 . We should add back a test for this at somepoint...

Reply:
hahaha